test_conformance/extensions/cl_khr_command_buffer/negative_tests/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+set(MODULE_NAME CL_KHR_COMMAND_BUFFER_NEGATIVE_TESTS)
+
+set(${MODULE_NAME}_SOURCES
+    main.cpp
+    command_buffer_create.cpp
+    ../basic_command_buffer.cpp
+)
+
+set_gnulike_module_compile_flags("-Wno-sign-compare")
```

Comment:
Since we're adding new tests, it'd be better to keep them warning. Ideally we'd remove this line (which is probably just copied from the other command-buffer test CMAKE) and fix any sign compare warnings.